docs/Local/installMPS.md

```diff
@@ -71,17 +73,17 @@ Start the MPS, RPS, and Sample Web UI in three separate command line terminals.
 
 **To start the MPS:**
 
-1. Open a new Terminal or Command Prompt and navigate to the `mps` directory inside the `open-amt-cloud-toolkit` directory. 
-
-2. Start the MPS server. It may take approximately 2-3 minutes to start.
+1. Open a new Terminal or Command Prompt and navigate to the `mps` directory inside the `open-amt-cloud-toolkit` directory. Start the MPS server. It may take approximately 2-3 minutes to start.
 
     === "Linux"
         ``` bash
+        cd mps
```

Comment:
Do we have these together or separate? Do people remember to press enter the second time? I don't think we can add a return character to the second cmd to run it automatically since markdown parses out spaces and empty new lines. i.e.
1. Navigate ... blah blah
```
cd mps
```
2.  Start .... yada yada
=== "Linux"
```
npm run devx
```
=== "Win
....

Reply:
Actually, can you take a look at how I've resolved these?

Reply:
Your solution works! We might just want to alter the written instructions to have emphasis on them navigating to open-amt-cloud-toolkit directory rather than mps/rps/sample-ui so that way the cmd can execute.  

I'd say the same for RPS/Sample-UI and they need to drop the '../' or '..\' in the cmds or it will cause an error for them.  Using those implies they are backtracking out of 'mps' which is not the case, both situations have brand new terminals open.